xbmc/games/controllers/windows/GUIControllerList.cpp

```diff
@@ -75,8 +75,17 @@ void CGUIControllerList::Deinitialize(void)
   m_controllerButton = nullptr;
 }
 
-bool CGUIControllerList::Refresh(void)
+bool CGUIControllerList::Refresh(const std::string &controllerId)
```

Comment:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types
```suggestion
bool CGUIControllerList::Refresh(const std::string& controllerId)
```

Reply:
https://github.com/xbmc/xbmc/pull/15763/commits

I assumed I squashed the commits a long time ago, but it looks like I was wrong.